lib/trento/domain/sap_system/sap_system.ex

```diff
@@ -287,6 +276,24 @@ defmodule Trento.Domain.SapSystem do
     |> Multi.execute(&maybe_emit_sap_system_tombstoned_event/1)
   end
 
+  def execute(
```

Comment:
Should we have other `execute` for the `deregistered_at` of the database?
The reality is that we don't have commands in the sap system aggregate outside of the registration and deregistration, so nothing can go wrong in reality.
So by now, it will simply work hehe

Reply:
Other than rollup, we don't have commands right know, but anyway I think the clause is sufficient